src/layout/dom.ts

```diff
@@ -270,13 +280,12 @@ function metadata(tag: string, id: number, parentId: number): void {
                 }
                 break;
             case "IFRAME":
-                value.metadata.boxmodel = true;
-                break;
-            default:
-                // Capture layout for any element with a user defined selector
-                value.metadata.boxmodel = value.selector.indexOf("*") === 0;
+                if (config.lean === false) { value.metadata.boxmodel = true; }
                 break;
         }
+
+        // Enable boxmodel if this node defines a new region
+        if (regionMap.has(nodes[id])) { value.metadata.boxmodel = true; }
```

Comment:
>regionMap [](start = 12, length = 9)

Does this mean we'll track box model only for specified nodes?
Also is the tracking recursive so we track the child if the parent is tracked?

Reply:
Yes, only specific nodes. No, it is not recursive and the key point here is that we don't have to track the child. It will capture the region and that's it.

Reply:
What does it mean to track a parent node and not its children?

---
In reply to: [395788883](https://github.com/microsoft/clarity-js/pull/170#discussion_r395788883) [](ancestors = 395788883)